geometry_msgs/Pose goal_pose
geometry_msgs/Vector3[] poses